messageService/message.go

```diff
@@ -13,6 +14,16 @@ type Authentification struct {
 	NickName string
 }
 
+//Message for online contacts after login
+type Contact struct {
```

Comment:
give me the link to yours contacts struct pls, i merged with master and  haven`t seen this  needed  struct for me